connector/spanmetricsconnector/internal/metrics/metrics.go

```diff
@@ -259,7 +265,10 @@ func (m *SumMetrics) GetOrCreate(key Key, attributes pcommon.Map) *Sum {
 	return s
 }
 
-func (s *Sum) AddExemplar(traceID pcommon.TraceID, spanID pcommon.SpanID, value float64) {
+func (s *Sum) AddExemplar(traceID pcommon.TraceID, spanID pcommon.SpanID, value float64, maxCount *int) {
```

Comment:
It feels a little odd to pass `maxCount` as a method parameter, instead of configuring it into the `Sum` property, given it should never change throughout its lifetime (statically configured).

What do you think about passing it into `NewSumMetrics` via `getOrCreateResourceMetrics`? Similarly for `NewExponentialHistogramMetrics` and `NewExplicitHistogramMetrics`.

Reply:
Done